api/v2/api.go

```diff
@@ -433,6 +433,10 @@ func (api *API) setupRoutes() error {
 			eth.POST("/request", api.RequestSignedPaymentMessage)
 			eth.POST("/confirm", api.ConfirmETHPayment)
 		}
+		stripe := payments.Group("/stripe")
+		{
+			stripe.POST("/charge", api.stripeCharge)
```

Comment:
is there a reason where the payments are:

* dash -> create
* eth -> request
* stripe -> charge

why not:

```
/payments/charge/${method}
```

Reply:
Yes:

ethereum for example is a two step process:
1) you need to `request` a signed message
2) submit the signed message to a smart contract and `confirm` it

dash involves `create`ing a payment forward with https://chainrider.io/ in the future dash payment methods will allow for reusable payment forward ids, which may end up having a call like `dash/reuse`.

Additionally I want to have the payment methods grouped by type, as each method might have different management/usability functions which aren't necessary related to different ones. For example with stripe you are placing a `charge` on a credit card.

Reply:
I see - it seemed like a missed opportunity to build an abstraction, but I suppose if we're aiming to keep it low level then this is alright

Reply:
Depending on how the payment system evolves it could definitely be revamped in the future when an eventual breaking change has to happen.